dependent/geology.csl

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
+  <info>
+    <title>Geology</title>
+    <id>http://www.zotero.org/styles/geology</id>
+    <link href="http://www.zotero.org/styles/geology" rel="self"/>
+    <link href="http://www.zotero.org/styles/the-geological-society-of-america" rel="independent-parent"/>
+    <link href="http://geosociety.org/GSA/Publications/Journals/Geology/GSA/Pubs/geology/home.aspx" rel="documentation"/>
+    <category citation-format="author-date"/>
+    <category field="geology"/>
+    <issn>0091-7613</issn>
+    <eissn>1943-2682</eissn>
+    <updated>2017-03-26T22:06:38+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
```

Comment:
@damnation333, missing `</style>` end tag here. The other two errors are complaints that the "template" link in two other styles are now orphaned (you can change those to point to "http://www.zotero.org/styles/the-geological-society-of-america" instead).